hawc/apps/riskofbias/exports.py

```diff
@@ -13,6 +13,7 @@ def _get_header_row(self):
         header = []
         header.extend(Study.flat_complete_header_row())
         header.extend(models.RiskOfBiasScore.flat_complete_header_row())
```

Comment:
why not just add these two items to the end of `models.RiskOfBiasScore.flat_complete_header_row`?